README.md

```diff
@@ -77,5 +77,14 @@ Snapshot testing is available, see documentation here: http://facebook.github.io
 You should commit snapshots to version control.
 
 
+#### Troubleshooting
+
+If you run into any issues due to existing installations, try running the following commands & then starting again.
+
+```
+cd ~/opt/nodes/6.3.0/lib/node_modules
```

Comment:
@williamhgough this is pretty specific to our setup, is there an npm way of doing this?

Reply:
@rhodimus aaaah yes, forgot this wasn't just an internal thing. Not sure if there is a less specific way but i'll look into it 👍